os/drivers/lcd/Kconfig

```diff
@@ -145,6 +145,25 @@ config LCD_ST7785
 	---help---
 		Enable LCD Driver, ST7785 with mipi.
 
+if LCD_ST7785
+choice
+	prompt "ST7785 LCD TYPE"
+	default LCD_ST7785_TYPEA
+	---help---
+		Select type of LCD.
+
+config LCD_ST7785_TYPEA
+	bool "ST7785 Type A LCD Driver"
+	---help---
+		Enable LCD Driver, ST7785 with mipi.
+
+config LCD_ST7785_TYPEB
+	bool "ST7785 Type B LCD Driver"
+	---help---
+		Enable LCD Driver, ST7785 with mipi.
+endchoice
+endif
```

Comment:
Let's move out of choice prompt "Choose LCD".

And if we select both A B, please use just config, not choice.